platformio.ini

```diff
@@ -35,10 +35,10 @@ custom_unpack_dir           = unpacked_littlefs
 build_unflags               = ${core.build_unflags}
 build_flags                 = ${core.build_flags}
 monitor_speed               = 115200
-monitor_port                = COM5
+; monitor_port                = COM5

```

Comment:
Why not remove?

Reply:
i dunno, cause mac might have some objections

Reply:
It is only a comment.